pkg/acsengine/defaults.go

```diff
@@ -740,6 +740,11 @@ func setStorageDefaults(a *api.Properties) {
 }
 
 func openShiftSetDefaultCerts(a *api.Properties) (bool, error) {
+	if len(a.OrchestratorProfile.OpenShiftConfig.ConfigBundles["master"]) > 0 &&
```

Comment:
wish there was a clearer indicator that this is a non-generate command

Reply:
I think the key here is "Default"